### PR TITLE
Feature/configuration

### LINKED ISSUE #42: Receive a configuration after authentication
after authentication the server should send some sort of confirutation object to allow the mod to opt into features.

@Ekwav can define the format.
this could be used to enable and disable other/future features in the mod using a web UI.

(e.g. ctrl+hover over item, etc.)

### PR DESCRIPTION
**Pull request by ToruiDev**
closes #42

### CODE REVIEW

File: src/main/java/de/torui/coflsky/CoflSky.java [hunk 1]
```diff
@@ -39,7 +40,7 @@
     @EventHandler
     public void init(FMLInitializationEvent event) throws URISyntaxException
     {
-    	
+
```
Comment (Ekwav) [inline src/main/java/de/torui/coflsky/CoflSky.java:43]: what purpose does this serve?

File: src/main/java/de/torui/coflsky/configuration/ConfigurationManager.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		} catch (IllegalAccessException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+	}
+
+	public boolean CompareProperties(Configuration old, Configuration newConfiguration)
+			throws IllegalArgumentException, IllegalAccessException {
```
Comment (Ekwav) [inline src/main/java/de/torui/coflsky/configuration/ConfigurationManager.java:45]: please align intendation

File: src/main/java/de/torui/coflsky/configuration/ConfigurationManager.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+	}
+
+	public void UpdatedProperty(Field propertyName) {
+		Description description = propertyName.getAnnotation(Description.class);
+		IChatComponent comp;
+		if (description != null) {
+
+			comp = new ChatComponentText("The Configuration Setting ")
+					.setChatStyle(new ChatStyle().setColor(EnumChatFormatting.BLUE))
+					.appendSibling(new ChatComponentText(description.value())
```
Comment (Ekwav) [inline src/main/java/de/torui/coflsky/configuration/ConfigurationManager.java:93]: the description is quite long, should't always the key/ a short name be used in the message and the description be put into the onHover?

**Review by Ekwav — CHANGES_REQUESTED** [review]

File: src/main/java/de/torui/coflsky/CoflSky.java (hunk 1 above)
Comment (ToruiDev) [reply src/main/java/de/torui/coflsky/CoflSky.java]: None, removed the empty line

File: src/main/java/de/torui/coflsky/configuration/ConfigurationManager.java (hunk 2 above)
Comment (ToruiDev) [reply src/main/java/de/torui/coflsky/configuration/ConfigurationManager.java]: i ran "format code" over the file

File: src/main/java/de/torui/coflsky/configuration/ConfigurationManager.java (hunk 3 above)
Comment (ToruiDev) [reply src/main/java/de/torui/coflsky/configuration/ConfigurationManager.java]: is now used where a description is provided

**Comment by ToruiDev** [comment]
i also fixed some imports with TimerCountdown that Intellij detected as incorrect while it was working.

**Pull request merged**